lib/worker_logs.js

```diff
@@ -1,6 +1,6 @@
-var fs = requrie('fs');
+var fs = require('fs');
```

Comment:
Given that require was spelled incorrectly here, is anything actually using this file? Can we just remove it? Nobody else has it required.